docs_src/examples/Ch-ExamplesAddingMQTTDevice.md

```diff
@@ -16,532 +16,390 @@ that implements the MQTT protocol versions 5.0, 3.1.1 and 3.1.
 
 Run Mosquitto using the following docker command:
 
-    docker run -d --rm --name broker -p 1883:1883 eclipse-mosquitto
+    docker run -d --rm --name broker -p 1883:1883 eclipse-mosquitto:1.6
 
 ## Run an MQTT Device Simulator
 
-![MQTT Device Service](MQTTDeviceService.png)
+![MQTT Device Service](EdgeX_ExamplesMQTTDeviceSimulator.png)
 
 This simulator has three behaviors:
 
-1.  Publish random number data every 15 seconds
-2.  Receive the reading request, then return the response
-3.  Receive the put request, then change the device value
-
-We created the following script to simulate the MQTT device:
-
-    // mock-device.js
-    function getRandomFloat(min, max) {
-        return Math.random() * (max - min) + min;
+1.  Publish random number data every 15 seconds.
+    
+    The simulator publishes the data to the MQTT broker with topic `DataTopic` and the message is similar to the following:
+    ```
+    {"name":"MQTT-Test-Device", "cmd":"randnum", "method":"get", "randnum":4161.3549}
+    ```
+    
+2.  Receive the reading request, then return the response.
+
+    1. The simulator receives the request from the MQTT broker, the topic is `CommandTopic` and the message is similar to the following:
+        ```   
+        {"cmd":"randnum", "method":"get", "uuid":"293d7a00-66e1-4374-ace0-07520103c95f"}
+        ```
+    2. The simulator returns the response to the MQTT broker, the topic is `ResponseTopic` and the message is similar to the following:
+        ```   
+        `{"cmd":"randnum", "method":"get", "uuid":"293d7a00-66e1-4374-ace0-07520103c95f", "randnum":42.0}
+        ```
+3.  Receive the put request, then change the device value.
```

Comment:
Receive the **set command**, then change the device value

Reply:
Modified.